### ISSUE TITLE
Allow RRULE property on Event component

### ISSUE THREAD

**Issue by ankurk91** [spatie/icalendar-generator#6]
Hi,

Thanks for the awesome package.

The only piece is missing in this package is `RRULE`, like:
```
RRULE:FREQ=DAILY;INTERVAL=10;COUNT=5
```
https://icalendar.org/iCalendar-RFC-5545/3-8-5-3-recurrence-rule.html

There is no need of full RRULE builder class, just add a  method to set the RRULE text property to event component.
I am using this package to generate RRULE string
https://github.com/simshaun/recurr

Thanks

**Comment by rubenvanassche** [comment]
Hi @ankurk91,

I've just added the possibility to define your own properties on components(https://github.com/spatie/icalendar-generator#extending-the-package).  In the future I would love to support RRULE's but then I would want to add it into the package without extra dependencies. At the moment I don't have enough time to build this but I've added it to my list 😄  I hope this solution will solve your problem for now.
